collector/collector.go

```diff
@@ -541,7 +544,7 @@ func (c collector) Collect(ch chan<- prometheus.Metric) {
 // only return metric if current job status is in progress
 func (c collector) checkConditionStatusPrint(result *MoonrakerHistoryCurrentPrintResponse, value float64) float64 {
 	var valueToReturn float64 = 0
-	if result.Result.Jobs[0].Status == "in_progress" {
+	if len(result.Result.Jobs >= 1) && result.Result.Jobs[0].Status == "in_progress" {
```

Comment:
closing bracket in wrong location, this need to be `len(result.Result.Jobs) >= 1`